Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -552,10 +575,19 @@ private static String msgForSaveSuccess(String ofType, Long ofId) {
         return MessageFormat.format("Save {0} successful.", ofType.toLowerCase());
     }
 
+    private static String msgForSaveUnsuccess(String ofType, Long ofId) {
+        return MessageFormat.format("Save {0} unsuccessful. Please refresh your browser and try again.",
```

Comment:
Optional: Can we reword the message?

Reply:
Believe the consensus was to say "Operation Unsuccessful" for each CRUD operation.